### PR TITLE
Frontend 1

### PR DESCRIPTION
**Pull request by asimism**
@mosic

### CODE REVIEW

File: app/assets/javascripts/cameras/single/archives.js.coffee [hunk 1]
```diff
@@ -35,14 +35,16 @@ initializeArchivesDataTable = ->
     initComplete: (settings, json) ->
       $("#archives-table_length").hide()
       if json.archives.length is 0
-        $(".dataTables_empty").text("There are no clips.")
+        #$(".dataTables_empty").text("There are no clips.")
```
Comment (mosic) [inline app/assets/javascripts/cameras/single/archives.js.coffee]: Remove commented out code. There are a couple of lines like this one in the diff.

File: app/views/cameras/single.html.erb [hunk 2]
```diff
@@ -33,12 +33,12 @@
...
 
-            <li>
+            <!--<li>
               <a data-toggle="tab" data-target="#apps" class="nav-tab-apps">Apps</a>
-            </li>
+            </li>-->
 
             <% if @camera['vendor_name'] == 'Hikvision Digital Technology' %>
-              <li>
+              <li>65
```
Comment (mosic) [inline app/views/cameras/single.html.erb]: Revert this "65" here.

**Comment by mosic** [comment]
Thanks @asimism. Please remove the commented/unused code and update the pull request.

File: app/views/widgets/live_view_private_widget.html [hunk 3]
```diff
@@ -16,6 +16,9 @@
     .video-js {
       padding-top: 56.2%;
     }
+    .video-js.vjs-fullscreen{
+        /*padding-top: 0;*/
+    }
```
Comment (mosic) [inline app/views/widgets/live_view_private_widget.html]: This block seems unnecessary? Unless you mistakenly commented out padding line.

**Comment by mosic** [comment]
@asimism I forgot, you need to sign the CLA before we can merge this.

It takes less than a minute, just go to https://www.clahub.com/agreements/evercam/evercam-dashboard

**Comment by marcoherbst** [comment]
Override that if you can milos

**Comment by mosic** [comment]
@marcoherbst I meant that not as a technical block, just as a thing that we do. Pull request can be merged either way. There's no reason for Asim not to do it though, and there's no way for me to do it instead of him (it uses the GitHub account for authentication).

Btw that's not the main reason I haven't merged this, the comment I left on the code is.

**Pull request merged**